docs/modules/configuring/con-configuring-vc-other-settings.adoc

```diff
@@ -0,0 +1,25 @@
+[id='ref-configuring-vc-other-settings-{context}']
```

Comment:
```suggestion
[id='ref-configuring-vc-logging-{context}']
```

Reply:
I would be tempted to keep this to logging only and change file name accordingly.

Reply:
Assuming this should be distinct from the **Setting logs** section we have in Monitoring proxies, maybe we should rename that section?